README.md

```diff
@@ -21,6 +21,8 @@ The data is split into two files, for DC and Marvel, respectively: dc-wikia-data
 
 `page_id, name, urlslug, ID, ALIGN, EYE, HAIR, SEX, GSM, ALIVE, APPEARANCES, FIRST APPEARANCE, YEAR`
 
+![Project Overview](img/project_overview_00.png)
```

Comment:
I would just note that for descriptive material we should say "alignment" instead of the column value `align`, otherwise looks good.